packages/teams-js/src/internal/utils.ts

```diff
@@ -528,3 +528,17 @@ export function validateUuid(id: string | undefined | null): void {
     throw new Error('id must be a valid UUID');
   }
 }
+
+/**
+ * Cache if performance timers are available to avoid redoing this on each function call.
+ */
+const supportsPerformanceTimers = !!performance && 'now' in performance;
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ * @returns current timestamp in milliseconds
+ */
+export function getCurrentTimestamp(): number {
+  return supportsPerformanceTimers ? performance.now() + performance.timeOrigin : new Date().getTime();
```

Comment:
on what basis is this feature flag set?
Since you are using a completely new field in the messageObject for monotonic timestamp, why is this flag needed?

Reply:
This flag is needed in case the app is running in an environment that `performance` timers are not yet available. Like old versions of chrome, internet explorer and stuff like that... Mostly it shouldn't present any issues but is just a safeguard for edge cases.